operations/helm/charts/mimir-distributed/CHANGELOG.md

```diff
@@ -30,6 +30,7 @@ Entries should include a reference to the Pull Request that introduced the chang
 
 * [CHANGE] Changed max unavailable ingesters and store-gateways in a zone to 50. #5327
 * [CHANGE] Don't render PodSecurityPolicy on Kubernetes >=1.24. (was >= 1.25). This helps with upgrades between 1.24 and 1.25. To use a PSP in 1.24, toggle `rbac.forcePSPOnKubernetes124: true`. #5357
+[FEATURE] Allow for deploying keda autoscaling objects as part of the helm chart. #4687
```

Comment:
can you prepend this with a bullet `*`?

Reply:
```suggestion
* [FEATURE] Added experimental feature for deploying keda autoscaling objects as part of the helm chart for the components: distributor, querier, query-frontend and ruler. Requires metamonitoring, for more details on metamonitoring see the Helm chart documentation.  #4687
```

Reply:
We should also mention the limitation that currently only URL embedded basic auth method is supported and we might add further Keda auth methods later, see https://keda.sh/docs/2.11/scalers/prometheus/